velox/functions/sparksql/Arithmetic.h

```diff
@@ -229,4 +229,13 @@ struct Log1pFunction {
     return true;
   }
 };
+
+template <typename T>
+struct CotFunction {
+  template <typename TInput>
```

Comment:
Is this template parameter needed? Isn't it always 'double'?

Reply:
The input of this function can only be double, I just refer to the implementation code of other trigonometric functions. And I find "All simple function classes need to be templated" in [scalar-functions](https://facebookincubator.github.io/velox/develop/scalar-functions.html).

Reply:
Let's remove TInput template parameter. It is not needed. It would be nice to update other functions as well (in a separate PR).

Reply:
"All simple function classes need to be templated" refers to the struct.